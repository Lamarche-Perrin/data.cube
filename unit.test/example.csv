dim1,dim2,dim3,obs1
a2,b2,c1,3
a3,b2,c2,3
a4,b1,c1,5
a2,b2,c2,2
a3,b2,c1,6
a2,b3,c1,5
a4,b2,c1,6
a4,b3,c1,4
a4,b3,c2,4
a2,b3,c2,8
a1,b1,c1,4
a1,b2,c1,3
a2,b1,c2,8
