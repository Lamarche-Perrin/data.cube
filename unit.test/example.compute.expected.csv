dim1,dim2,dim3,obs1,obs2,exp1,dev1,exp2,dev2
a1,b1,c1,4,3,1.3770491803,0.1008799944,2.5416666667,2.1675486233
a4,b1,c1,5,6,3.737704918,0.0344078212,2.5416666667,3.1041518536
a2,b1,c2,8,5,3.5519125683,0.153626745,2.5416666667,6.6642171421
a1,b2,c1,3,4,1.3770491803,0.0552483155,2.5416666667,1.3808426893
a2,b2,c1,3,5,5.1147540984,-0.0378542154,2.5416666667,1.3808426893
a3,b2,c1,6,6,1.7704918033,0.173194656,2.5416666667,4.1746086438
a4,b2,c1,6,7,3.737704918,0.067161622,2.5416666667,4.1746086438
a2,b2,c2,2,6,3.5519125683,-0.0271670843,2.5416666667,-0.6289142983
a3,b2,c2,3,7,1.2295081967,0.0632892368,2.5416666667,1.3808426893
a2,b3,c1,5,6,5.1147540984,-0.0026833431,2.5416666667,3.1041518536
a4,b3,c1,4,8,3.737704918,0.0064162179,2.5416666667,2.1675486233
a2,b3,c2,8,7,3.5519125683,0.153626745,2.5416666667,6.6642171421
a4,b3,c2,4,9,2.5956284153,0.0409125334,2.5416666667,2.1675486233
